client/cli/src/commands/run_cmd.rs

```diff
@@ -100,18 +100,28 @@ pub struct RunCmd {
 	#[clap(long)]
 	pub unsafe_ws_external: bool,
 
-	/// Set the the maximum RPC payload size for both requests and responses (both http and ws), in
-	/// megabytes. Default is 15MiB.
+	/// DEPERECATED, this has no affect anymore. Use `rpc_max_request_size` or
```

Comment:
```suggestion
	/// DEPRECATED, this has no affect anymore. Use `rpc_max_request_size` or
```

Is there automation in place to show this message to end users?

Reply:
no unfortunately not what I'm aware of.